src/Domain/Donation.php

```diff
@@ -215,11 +215,9 @@ public function toHookModel(): array
         $data['updatedTime'] = $this->getUpdatedDate()->format(DateTime::ATOM);
         $data['amountMatchedByChampionFunds'] = (float) $this->getConfirmedChampionWithdrawalTotal();
         $data['amountMatchedByPledges'] = (float) $this->getConfirmedPledgeWithdrawalTotal();
-        $data['optInCharityEmail'] = $this->getCharityComms();
 
-        // For backwards compatability with Enthuse
-        if ($this->getPsp() != 'stripe') {
-            unset($data['optInCharityEmail']);
+        if ($this->getCharityComms) {
```

Comment:
@tbg-jasonfung this (a) must be a method call – I suggest using e.g. the free Jetbrains IDEs to highlight errors like this – and (b) must check for null as distinct from false, otherwise opt outs will be ignored.

Reply:
Actually @NoelLH shouldn't we just set it regardless what the data is, SF side defaults `null` and uses what's currently saved on the record, otherwise for Stripe stuff it will set to T or F.